packages/button/src/LionButton.js

```diff
@@ -8,6 +8,14 @@ export class LionButton extends DelegateMixin(SlotMixin(LionLitElement)) {
         type: Boolean,
         reflect: true,
       },
+      role: {
```

Comment:
Why do we want to allow our users to override the role of lion-button to be anything other than button?

Reply:
imho we should provide good defaults... but we should not enforce these defaults... if anyone wants to do something stupid or needs to apply a workaround for something... we should not prevent it...  

in probably 99% of the cases no one will set it themselves... so its a good default and users generally do not need to know/care about it... but if they do they still can